README.md

```diff
@@ -1,75 +1,159 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
 
-# Project Name
+<div align="center">
+  
+  
+  <h1>Hello React Front-end App</h1>
+</div>
 
-> Description the project.
+# 📗 Table of Contents
 
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Future features](#future-features)
+    - [Related repos](#related-repos)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-## Built With
+# 📖 Hello React front-end <a name="about-project"></a>
 
-- Major languages
-- Frameworks
-- Technologies used
+> This is a practise project that demostrates how to create a full stack application using two different repositories. React and redux for front-end. And Ruby on rails for backend.
 
-## Live Demo (if available)
+**Hello React front-end**
 
-[Live Demo Link](https://livedemo.com)
+## 🛠 Built With <a name="built-with"></a>
 
+### Tech Stack <a name="tech-stack"></a>
 
-## Getting Started
+- JavaScript
+- React-DOM
+- React
+- Redux Toolkit
+- Redux
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+> JavaScript and React and redux was used to develop the front-end of this application which is available in this app. Postgres and Ruby on Rails was used to develop the back-end which is in a separate app.
 
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
 
-To get a local copy up and running follow these simple example steps.
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://nodejs.com/">node.js</a></li>
+  </ul>
+</details>
```

Comment:
- [OPTIONAL] Let's remove node.js from the Server Tech section since we did not use it in the repo 🚮 
![image](https://user-images.githubusercontent.com/79058364/233806391-4597104d-06eb-46d9-8608-59dbaf66679d.png)